Testreglar/3.3.1/App/app-3.3.1b.json

```diff
@@ -0,0 +1,276 @@
+{
+	"namn": "App-3.3.1b Skjema gir feilmelding hvis feil inndata blir oppdaget automatisk 2023",
+	"id": "app-3.3.1b",
+	"testlabId": 430,
+	"versjon": "1.0",
+	"type": "App",
+	"spraak": "nb",
+	"kravTilSamsvar": "<p>For skjemaelementer der inndatafeil blir oppdaget automatisk, er følgende punkter oppfylt:</p>\r\n<ul>\r\n<li>Skjemaelementet der feilen oppstod identifiseres og</li>\r\n<li>Brukeren får en presis tekstlig beskrivelse av feilen</li>\r\n<li>Feilmeldingen er kodet som tekst og</li>\r\n<li>Informasjonen blir liggende i skjemaet, med mindre informasjonen er knyttet til personvern eller sikkerhet</li>\r\n</ul>",
+	"side": "2.1",
+	"element": "3.1",
+	"kolonner": [
+		{
+			"title": "2.2"
+		},
+		{
+			"title": "2.3"
+		},
+		{
+			"title": "2.4"
+		},
+		{
+			"title": "2.5"
+		},
+		{
+			"title": "2.6"
+		},
+		{
+			"title": "2.7"
+		},
+		{
+			"title": "3.2"
+		},
+		{
+			"title": "3.3"
+		},
+		{
+			"title": "3.4"
+		},
+		{
+			"title": "3.5"
+		},
+		{
+			"title": "3.6"
+		}
+	],
+	"steg": [
+		{
+			"stegnr": "2.1",
+			"spm": "Hvilken appside tester du?",
+			"ht": "<p>Beskriv appsiden med få stikkord, eller velg i listen.</p>",
+			"type": "tekst",
+			"label": "Appside:",
+			"datalist": "Sideutvalg",
+			"oblig": true,
+			"ruting": {
+				"alle": {
+					"type": "gaaTil",
+					"steg": "2.2"
+				}
+			}
+		},
+		{
+			"stegnr": "2.2",
+			"spm": "Har appsiden digitale skjema/skjemaelement?",
+			"ht": "<p><strong>Merk: </strong>Du skal ikke teste skjema i PDF, Word eller lignende.</p>",
```

Comment:
er pdf og word-skjema relevant for app?

Reply:
De finnes i alle fall i feks  nettbank-apper og digital postkasse apper.